### PR TITLE
org and individual registration screens

### PR DESCRIPTION
**Pull request by shaala**
#### Short description of what this resolves:
Template of Individual and Organization registration screen

### CODE REVIEW

File: src/pages/login/login.ts [hunk 1]
```diff
@@ -1,65 +1,24 @@
...
-          message: this.loginErrorString,
-          duration: 3000,
-          position: 'top'
-        });
-        toast.present();
-      });
-      /*this.auth.login('linkedin').then(() => this.navCtrl.push(MainPage))
-      .catch( e => this.navCtrl.setRoot('WelcomePage'));*/
+  
+      this.navCtrl.push('IndregisterPage');
```
Comment (lindsgray) [inline src/pages/login/login.ts:20]: okay a couple questions here:
your info for the login page is hosted in the welcome page therefore the welcome.ts, yeah? and if they are logging in, why is this redirecting to the register page?

File: src/pages/indregister/indregister.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<ion-header>
+
+  <ion-navbar>
+    <ion-title>Individual Registraion Form</ion-title>
```
Comment (lindsgray) [inline src/pages/indregister/indregister.html:4]: *Registration

File: src/pages/indregister/indregister.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+export class IndregisterPage {
+ 
+  constructor(public navCtrl: NavController) { }
+  // Our translated text strings 
+  
+
+  
+  login() {
+  
+      this.navCtrl.push('WelcomePage');
```
Comment (lindsgray) [inline src/pages/indregister/indregister.ts:20]: you need to import and push to the resource page. when you push to the welcome, that is where you hosted the login information, in my opinion we should have this redirect into the app as opposed to back out to the login page where they need to login.

File: src/pages/login/login.html [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <ion-header>
 
   <ion-navbar>
-    <ion-title>{{ 'LOGIN_TITLE' | translate }}</ion-title>
+    <ion-title>Organization Registraion Form</ion-title>
```
Comment (lindsgray) [inline src/pages/login/login.html:4]: *Registration

File: src/pages/login/login.html [hunk 5]
```diff
@@ -10,28 +10,38 @@
 <ion-content>
   <form (submit)="login()">
     <ion-list>
+      <ion-item>
+        <ion-label fixed>Org Name</ion-label>
```
Comment (lindsgray) [inline src/pages/login/login.html:14]: This should say "Name of Organization" to be more professional

File: src/pages/login/login.html [hunk 6]
```diff
@@ -10,28 +10,38 @@
...
-        <ion-input type="email" name="email"></ion-input>
+        <ion-label fixed>Tax Id </ion-label>
+        <ion-input type="text" name="taxId"></ion-input>
       </ion-item>
 
-      <!--
-      Want to use a Username instead of an Email? Here you go:
+      <ion-item>
+        <ion-label fixed>Address </ion-label>
+        <ion-input type="text" name="orgAddress"></ion-input>
```
Comment (lindsgray) [inline src/pages/login/login.html:25]: I would break this down even more into "street address" "city" "state" "zip code"

**Review by lindsgray — CHANGES_REQUESTED** [review]

**Comment by ChristaFox** [comment]
I didn’t think we were using linked in to log in any more, and the address field is broken down more in the database as well.

**Comment by ChristaFox** [comment]
Here is the db for reference of fields :

**Pull request merged**